as/cNFT/assembly/models/persistent_nft_contract_metadata.ts

```diff
@@ -26,6 +27,11 @@ export class NFTContractExtra {
     max_copies: u32
     default_max_len_payout: u32
     mints_per_address: u32
+    mint_payee_id: ''
+    mint_royalty: {
+        id: AccountId
+        amount: u32
+    }
```

Comment:
This should be like below I think. Otherwise we are getting an error for `yarn test`.

```
@nearBindgen
export class MintRoyalty {
    id: AccountId
    amount: u32
}

@nearBindgen
export class NFTContractExtra {
    mint_price: string
    max_copies: u32
    default_max_len_payout: u32
    mints_per_address: u32
    mint_payee_id: AccountId
    mint_royalty: MintRoyalty
}

```